### ISSUE TITLE
[ask] - esx_banking - trigger log

### ISSUE THREAD

**Issue by amgmacht** [esx-framework/esx-legacy#669]
A possibility to set a trigger event that if you get salary or unemployment benefits that these are also displayed in the bank logs in the bank?

**Comment by Gellipapa** [comment]
Hi! I can write export but you have to call the export where you want it. But it's not a BUG, it's a feature :)

**Comment by amgmacht** [comment]
Yes I know I didn't know how to make this a feature sry but that would be super cool if you could do it ^^

**Comment by amgmacht** [comment]
Can you ping me when you've done the export ^^ ?

**Comment by Gellipapa** [comment]
Hi! #708 add this feature. Thanks for your patience! @amgmacht

**Comment by amgmacht** [comment]
@Gellipapa would it be possible to specify a purpose?

for example:
salary
car purchase
Steer

Instead of just deposit or withdraw?

**Comment by Gellipapa** [comment]
It requires more rewriting on the frontend side because it needs to be displayed everywhere. If you just look at the table it's easy, but if you need to go into the history it requires a bigger rewrite, I'll see how much I can do. @amgmacht
